deps/src/main/java/com/microsoft/azure/sdk/iot/deps/ws/impl/WebSocketUpgrade.java

```diff
@@ -314,14 +314,7 @@ public Boolean validateUpgradeReply(byte[] responseBytes)
 
         scanner.close();
 
-        if ((isStatusLineOk) && (isUpgradeHeaderOk) && (isConnectionHeaderOk) && (isProtocolHeaderOk) && (isAcceptHeaderOk))
-        {
-            return true;
-        }
-        else
-        {
-            return false;
-        }
+        return (isStatusLineOk) && (isUpgradeHeaderOk) && (isConnectionHeaderOk) && (isProtocolHeaderOk) && (isAcceptHeaderOk);
```

Comment:
I don't think we need the parenthesis here, right?

Reply:
correct! I'll remove these